composer.json

```diff
@@ -31,6 +34,9 @@
         "phing/phing": "^2.5",
         "sebastian/phpcpd": "^3",
         "wimg/php-compatibility": "^8",
-        "dealerdirect/phpcodesniffer-composer-installer": "^0.4"
+        "dealerdirect/phpcodesniffer-composer-installer": "^0.4",
+        "php-http/httplug": "2.x-dev@dev",
+        "php-http/guzzle6-adapter": "dev-patch-psr18@dev",
```

Comment:
I need to think about. This is only for tests, so maybe I can prepare a PR for adapter supporting 7.0 so we can use it instead?

I guess you have a good reason to support 7.0 because it's reaching EOL soon.

I totally missed that during development.

Reply:
Looks like guzzle itself will end with php 7.0 support, but PR is very wip: https://github.com/guzzle/guzzle/pull/2122/files

Reply:
Sorry, but supporting 7.0 is our bussiness requirement.